src/gurt/dlog.c

```diff
@@ -387,6 +387,77 @@ log_exceed_threshold(void)
 	return mst.log_size + mst.log_buf_nob >= mst.log_size_max;
 }
 
+static int
+log_rotate(void)
+{
+	int rc = 0;
+
+	/* exceeds the size threshold, rename the current log file
+	 * as backup, create a new log file.
+	 */
```

Comment:
This is a comment for the new function so goes on L389.

Reply:
Moved the comment to the function start